src/main/scala/is/hail/utils/richUtils/RichHadoopConfiguration.scala

```diff
@@ -29,7 +29,7 @@ class RichHadoopConfiguration(val hConf: hadoop.conf.Configuration) extends AnyV
       os
   }
 
-  private def open(filename: String): InputStream = {
+  def open(filename: String): InputStream = {
```

Comment:
Ah I didn't see `readFile` and friends. Doesn't it seem like classic non-orthoganlity to have a separate method for each type of resource rather than a `using`-like primitive?